src/App.css

```diff
@@ -203,7 +203,7 @@
 
 @media (max-width: 1200px) {
   .add-stonks {
-    margin: 100px 0 0 70px;
+    margin: 2rem 0 0 4.5rem;
```

Comment:
I was just talking with Becca about using rem vs px and learned that rem is often preferred for accessibility reasons!

Reply:
I actually didn't even know that about accessibility! I just changed it because designers tell me not to use pixels! :)